fingerprint2.js

```diff
@@ -209,6 +216,94 @@
       }
       return keys
     },
+    isFlashKey: function (keys){
```

Comment:
Please rename this to `hasFlash`

Reply:
Also, how do the flash and silverlight checks relate to the list of browser plugins that are already collected by fingerprintjs? Does this increase entropy? If so why not include all plugins as a separate component?

Reply:
908956a Rename Done